src/main/java/frc/trigon/robot/subsystems/arm/kablamaarm/KablamaArmConstants.java

```diff
@@ -0,0 +1,128 @@
+            ELEVATOR_ENCODER_ID = 6;
+    private static final CANSparkMax.IdleMode
+            ELEVATOR_IDLE_MODE = CANSparkMax.IdleMode.kBrake,
+            ANGLE_IDLE_MODE = CANSparkMax.IdleMode.kBrake;
+    private static final int VOLTAGE_COMPENSATION_SATURATION = 12;
+    private static final boolean
+            MASTER_ANGLE_INVERTED = false,
+            MASTER_ELEVATOR_INVERTED = false,
+            FOLLOWER_ANGLE_INVERTED = false,
+            FOLLOWER_ELEVATOR_INVERTED = false;
+    private static final SensorDirectionValue ANGLE_ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final boolean ELEVATOR_ENCODER_PHASE = false;
+    private static final int
+            ELEVATOR_ENCODER_OFFSET = 0,
+            ANGLE_ENCODER_OFFSET = 0;
+    private static final AbsoluteSensorRangeValue ANGLE_ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    static final WPI_TalonSRX ELEVATOR_ENCODER = new WPI_TalonSRX(ELEVATOR_ENCODER_ID);
+    static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    static final StatusSignal<Double>
+            ANGLE_MOTOR_POSITION_SIGNAL = ANGLE_ENCODER.getPosition(),
+            ANGLE_MOTOR_VELOCITY_SIGNAL = ANGLE_ENCODER.getVelocity();
+
+    private static final double
+            ANGLE_P = 5,
+            ANGLE_I = 1,
+            ANGLE_D = 1,
+            ELEVATOR_P = 5,
+            ELEVATOR_I = 1,
+            ELEVATOR_D = 1;
+    static final PIDController
+            ANGLE_PID_CONTROLLER = new PIDController(ANGLE_P, ANGLE_I, ANGLE_D),
+            ELEVATOR_PID_CONTROLLER = new PIDController(ELEVATOR_P, ELEVATOR_I, ELEVATOR_D);
+
+    private static final double
+            ANGLE_MOTOR_KS = 0,
+            ANGLE_MOTOR_KV = 0,
+            ANGLE_MOTOR_KA = 0,
+            ANGLE_MOTOR_KG = 0,
+            ELEVATOR_MOTOR_KS = 0,
+            ELEVATOR_MOTOR_KV = 0,
+            ELEVATOR_MOTOR_KA = 0,
+            ELEVATOR_MOTOR_KG = 0;
+    static final ArmFeedforward ANGLE_FEEDFORWARD = new ArmFeedforward(
+            ANGLE_MOTOR_KS, ANGLE_MOTOR_KG, ANGLE_MOTOR_KV, ANGLE_MOTOR_KA
+    );
+    static final ElevatorFeedforward ELEVATOR_FEEDFORWARD = new ElevatorFeedforward(
+            ELEVATOR_MOTOR_KS, ELEVATOR_MOTOR_KG, ELEVATOR_MOTOR_KV, ELEVATOR_MOTOR_KA
+    );
+    public static final double RETRACTED_ARM_LENGTH_METERS = 0.64;
+    public static final double THEORETICAL_METERS_PER_REVOLUTION = 0.1256;
```

Comment:
Why is `THEORETICAL_METERS_PER_REVOLUTION` public? And why did you move `RETRACTED_ARM_LENGTH_METERS` here?